openmc/plotter.py

```diff
@@ -61,12 +61,15 @@
 # Minimum and maximum energies for plotting (units of eV)
 _MIN_E = 1.e-5
 _MAX_E = 20.e6
+_MIN_E_PROTON = 1.e2
+_MAX_E_PROTON = 20.e8
+
```

Comment:
Just checking: are these the limits of the available data? Do the incident proton data have a lower limit of 100 eV? Is the upper limit 2 GeV? I note that the plots on the PR only go up to 200 MeV. Assuming that 2 GeV is the correct upper limit, 20.e8 seems like an odd way to write it. I can see the similarity with how they've specified _MAX_E as 20.e6 (20 MeV) but I can't see the sense in writing 20.e8 unless it's some pythonesque thing I'm missing. 2.e9 would make more sense to me.

Also, is there one too many blank lines here?

Reply:
I think @davidjohnlong just put these in from what he observed. They aren't necessarily the max or min seen in the library's. These limits don't really do much as I assume most of the time you would want to change your axis scales anyway. I'll change to 2.e9 though.